System-Testing-Deployments-and-Upgrades/User-Authorization/index.rst

```diff
@@ -0,0 +1,162 @@
+
+.. |author| replace:: *Tiago Ribeiro*
+.. |contributors| replace:: *Wouter van Reeven*, *Rob Bovill*
+
+.. _User-Authorization:
+
+##################
+User Authorization
+##################
+
+The Rubin Observatory Control System has an authorization system designed to help manage and coordinate user interaction with the system.
+
+By default, the system is designed to be operating under autonomous CSC control.
+For example, during survey operations the Scheduler will compute the observing queue and feed them to the ScriptQueue, which then executes the SAL Scripts required to observe the target.
+
+At the same time, users may want to interact with the system to perform engineering or other tasks that require a finer level of interaction with the system, which is usually done from :ref:`nublado <Control-System-User-Interfaces-Nublado>`.
+Nevertheless, before they can interact with the system, they need to request authorization to directly command the CSCs.
+Users can also block CSCs from interacting with other CSCs if they want to guarantee nothing else in the system will interfere with their activities.
+
+.. _User-Authorization-Prerequisites:
+
+Prerequisites
+=============
+
+* Requesting authorization happens as part of interacting with the Observatory Control System.
+  Before you start make sure you have :ref:`announced the component usage <Operational-Tutorials-Annouce-Component-Usage>`.
+
+* The authorization system may require an observing specialist or other administrator approval (in LOVE).
+  Make sure you coordinate with the observing specialists to approve your request beforehand.
+
+* If you are an observing specialist or administrator, see :ref:`Managing-User-Authorization`.
+  
+
+.. _User-Authorization-Post-Condition:
+
+Post-Condition
+==============
+
+* User is authorized to control CSCs.
+
+.. _User-Authorization-Tutorial-Steps:
+
+Tutorial Steps
+==============
+
+Using High-Level Control Classes
+--------------------------------
+
+In most cases, users interact with the system using one of the high-level control classes from `ts_observatory_control`_; ``MTCS``, ``ComCam``, etc.
+
+.. _ts_observatory_control: https://ts-observatory-control.lsst.io
+
+These classes provides utility methods to request and release authorization for all their components.
+If you are using any of the classes provided by `ts_observatory_control`_ you can request authorization by running the following command;
+
+.. code-block:: python
+
+  from lsst.ts import salobj
+  from lsst.ts.observatory.control.maintel import MTCS
+
+  domain = salobj.Domain()
+
+  mtcs = MTCS(domain)
+
+  await mtcs.start_task
+
+  # Request authorization to operate all MTCS components
+  await mtcs.request_authorization()
```

Comment:
What happens if this is denied? What are users to do? Especially if someone forgot to release it.

Reply:
In the "Prerequisites" session I mention:

```
* The authorization system may require an observing specialist or other administrator approval (in LOVE).
  Make sure you coordinate with the observing specialists to approve your request beforehand.
```

You can have more than one person authorized so someone forgetting to release it is not a problem. So I guess if the authorization is denied it means the users didn't met a prerequisite. 

Also, in the Troubleshooting session I add a couple failure scenarios.